src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,161 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int sizeOfList;
+    private Node firstNode;
+    private Node lastNode;
+
+    public MyLinkedList() {
+    }
```

Comment:
It's called default constructor - because it will be created by default if no other constructor is specified, you can safely remove it